config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/eks-controller
-  newTag: 1.3.3
+  newTag: 0.0.0-non-release-version
```

Comment:
Can you please pull all the eks-controller tags? maybe we can also release 1.3.4?

Reply:
I was trying to understand that, so I should just revert the changes (find/replace?) from `0.0.0-non-release-version` to `1.3.3`?  Or do I need to change something in the generator.yaml?

Reply:
So the code-generator, gets the tag from the `eks-controller` local repository tags and uses that the generate the release manifests. You can try `git pull --tags --all` in `eks-controller` repository and regenerate the code as usual

Reply:
If that doesn't work it could be a bug in the generation script. A workaround is to run something like `RELEASE_VERSION=v1.3.4 ./scripts/build-controller-release.sh eks`

Reply:
Gotcha, that worked, I just had to pull the tags from the upstream and re-run the generator.